### PR TITLE
test sum works

### PR DESCRIPTION
**Pull request by elsaalarcon**

### CODE REVIEW

File: spec/simple_arithmetic_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+describe 'retun 10 if is true' do
+  it "the result of sum is 10" do
+    sum = 5 + 5
+    expect(sum).to eq 10
+  end
+end
+  
+
+# require_relative '../one/print_name.rb'
```
Comment (geocodinglife) [inline spec/simple_arithmetic_spec.rb]: Remove unneeded code.

Comment (elsaalarcon) [reply spec/simple_arithmetic_spec.rb]: Is done.

File: spec/simple_arithmetic_spec.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require_relative '../one/simple_arithmetic.rb'
+
+describe 'retun 10 if is true' do
+  it "the result of sum is 10" do
+    sum = 5 + 5
+    expect(sum).to eq 10
+  end
+end
+
```
Comment (geocodinglife) [inline spec/simple_arithmetic_spec.rb]: Remove extra lines.

Comment (elsaalarcon) [reply spec/simple_arithmetic_spec.rb]: Done.

**Review by geocodinglife — APPROVED** [review]
Looks Good!

**Pull request merged**